gnovm/pkg/gnolang/go2gno.go

```diff
@@ -96,22 +98,34 @@ func MustParseExpr(expr string) Expr {
 	return x
 }
 
+// parseCost computes and returns the cost of parsing from the parser stats.
+func parseCost(stats *parser.Stats) types.Gas {
+	// The parsing cost is a function of the number of tokens and the
+	// nesting level reached during the parsing, which reflect both the size
+	// and the complexity of the AST.
+	return types.Gas(stats.NumTok * math.Ilogb(float64(1+stats.TopNest)))
+}
```

Comment:
The Gas is a measure of computation time  benchmarked in nano seconds. Can we benchmark this?